CONTRIBUTING.md

```diff
@@ -85,6 +85,35 @@ git push origin my-branch
 
 5. Open [this project on GitHub](https://github.com/theforeman/foreman-js), then click “Compare & pull request”.
 
+## Testing Pull Requests
+
+1. Open pull request in Github and go to the tasks tab.
+2. Expand `Build tarballs for testing` jobs and open the relevant action. For example, the job for the test package would be `build-tarball (test)`.
```

Comment:
The artifact seems to be available no matter which action I select in the nav bar.
```suggestion
2. Expand `Build tarballs for testing` and open any of the associated actions. For example, the job for the test package would be `build-tarball (test)`.
```